db/fixtures/miq_actions.csv

```diff
@@ -10,6 +10,7 @@ evm_event,Show EVM Event on Timeline
 script,Execute an external script
 prevent,Prevent current event from proceeding
 container_image_analyze,Initiate SmartState Analysis for Container Image
+container_image_scan_all,Sequentially scan all providers images.
```

Comment:
The description should also say "container" somewhere so it's not confused with VM images.